### ISSUE TITLE
Not able to navigate to some objects in developer portal

### ISSUE THREAD

**Issue by i02coroj** [Azure/api-management-developer-portal#1889]
## Bug description

When I navigate through the different objects defined in my yaml via Developer Portal some links don't work because some of them are not loaded.
For instance in next screenshot you can see an outboundInvoice object but in the developer portal the link to it doesn't work.
I've even added a "definitions" object (you also can see it in that screenshot) that explicitly references all objects used but has made no difference.
I have the feeling it has to do with inheritance: when there's no inheritance objects look to load correctly. But I might be wrong.

![image](https://user-images.githubusercontent.com/29977358/182820635-36e8cc68-d50a-44e7-b54f-e2e091a6776c.png)

## Reproduction steps

1. Sign in
2. Go to APIs list and select the only one available.
3. Click on 'PostInvoice' payload
4. Click on 'outboundInvoice' object
5. It doesn't show up.

## Expected behavior
outboundInvoice object details should be shown.

## Is your portal managed or self-hosted?
Managed

## API Management service name
ig-weu-dev-integration-apim

## Environment
 - Operating system: Windows 10 64 bits
 - Browser: Edge and Chrome
 - Version: Edge 103.0.1264.77. Chrome 103.0.5060.134

## Additional context
Add any other context about the problem here, including screenshots.

**Comment by i02coroj** [comment]
Interestingly I've spotted that if I grab the entire URL pointing to the object and use it directly it works.
So, for my example above with outboundInvoice object, if I open next link directly in a new tab:
https://ig-weu-dev-integration-apimanagement.developer.azure-api.net/api-details#api=gateway-api-services-v1&operation=post-postinvoice&definition=outboundInvoice

The outboundInvoice is correctly loaded.
Not sure if that helps.

**Comment by azaslonov** [comment]
@i02coroj, sorry, I can't reproduce it. For me it works all the time (in your portal).

**Comment by i02coroj** [comment]
Thanks for looking, @azaslonov .
Wow, that's surprising, it's failing not only for me but for many coworkers and clients so i don't get why it works for you.

Again, it only fails when I go step by step, if I use the entire URL pointing to the object it works.
Also, I'm signing in first and you are not, wondering it that can make any difference.

Here you've got a video when I reproduce the issue: https://tigplc-my.sharepoint.com/:v:/g/personal/josealberto_coronado_innovation_group/EQl7t_PWT0FDlR5iJCDXEmUByQXLwcfYc50Ug95sChG8bA?e=FOiL8o
(I've granted access to apimportalfeedback@microsoft.com and to the email you have in your profile. Hope it's ok.)

For signing in I've created a temporal user (no problems with security as it doesn't really make any difference).
User: azaslonov_testing@innovation.group
Pwd: azaslonov_testing

Thanks again for looking.

**Comment by JanEnevoldsen** [comment]
Hi @azaslonov,
I have exactly the same issue on our Developer Portal (https://developer.dsv.com)
It only works when using an Incognito window. 
I am able to make it fail in the incognito window as well if I switch to another API by using the dropdown menu.

**Comment by i02coroj** [comment]
In my case I've made one more finding: it always work for the endpoint shown by default.
So, in my case when I load the list of endpoints I see this: 
![image](https://user-images.githubusercontent.com/29977358/183086229-a3208164-feff-4e21-817b-e625a5174ac1.png)

Every single object in that multiTypeRequest endpoint works smoothly, everything loaded. However I click in any other endpoint it doesn't. Actually if I go back to multitypeRequest endpoint it doesn't work anymore.
If I go ahead and update my yaml so a different endpoint shows up first that one works but the rest don't.

So, in summary; only the one in the top of the list works ok, and only the first time it's loaded.

**Comment by azaslonov** [comment]
Thanks folks, let us play more with it, I'll keep you posted.

**Comment by kcyk01** [comment]
It looks like it's tied to the caching, so first load of an API will always work while subsequent loads from refreshing or navigating to other operations (in the same API) fail. Also explains why we are seeing first-time loading of an API in Incognito fine until it gets saved in the IndexedDB for subsequent requests.

Clearing the `keyval` in the IndexedDB seems to temporarily fix this as it causes the page to reload the JSON schema and render properly before it's cached again.

Possibly related, but also getting this in Firefox (102.0.1) Private Browsing in the Console:
```
Uncaught (in promise) DOMException: A mutation operation was attempted on a database that did not allow mutations. 
```

**Comment by i02coroj** [comment]
Hi @azaslonov , so at the light of @kcyk01 's comment, do you think it's feasible to fix the issue soon-ish?
This is blocking us (and I assume many other people as @JanEnevoldsen ) quite a lot.

Thanks.

**Comment by azaslonov** [comment]
@i02coroj, @kcyk01, @JanEnevoldsen, the hotfix is being rolled out right now. It should be complete in every region by end of this week. Apologies for all the troubles.

**Comment by JanEnevoldsen** [comment]
Hi @azaslonov, I can confirm that our Developer Portal is again working as it should. Thank you.
@i02coroj, thank you for having reported this issue.

**Comment by i02coroj** [comment]
Lovely, it works smoothly for me now too.
Thanks @azaslonov for looking into it.

**Comment by azaslonov** [comment]
Thanks for confirmation folks! Closing the issue as resolved.
